src/server/project.ts

```diff
@@ -744,12 +746,10 @@ namespace ts.server {
                 clearMap(this.missingFilesMap, closeFileWatcher);
                 this.missingFilesMap = undefined!;
             }
-            if (this.packageJsonFilesMap) {
-                clearMap(this.packageJsonFilesMap, closeFileWatcher);
-                this.packageJsonFilesMap = undefined;
-            }
             this.clearGeneratedFileWatch();
             this.clearInvalidateResolutionOfFailedLookupTimer();
+            this.autoImportProviderHost?.close();

```

Comment:
Sorry i forgot to mention yesterday but you need to close and release the program even from disableLanguageService